paasta_tools/utils.py

```diff
@@ -1604,5 +1605,4 @@ def is_deploy_step(step):
     Returns true if the given step deploys to an instancename
     Returns false if the step is a predefined step-type, e.g. itest or command-*
     """
-    return not ((step in DEPLOY_PIPELINE_NON_DEPLOY_STEPS)
-                or (step.startswith('command-')))
+    return not ((step in DEPLOY_PIPELINE_NON_DEPLOY_STEPS) or (step.startswith('command-')))
```

Comment:
The split line didn't pass flake8 for me, not sure how it got committed / how general_itests haven't been failing.